cloud/performance.rst

```diff
@@ -88,3 +88,24 @@ To measure the number of active connections at a given time, run:
   SELECT COUNT(*)
     FROM pg_stat_activity
    WHERE state <> 'idle';
+
+PostgreSQL/PgBouncer Configuration
+==================================
+
+In addition to adding nodes and scaling their hardware, the Cloud Console allows changing database server configuration parameters. Adjusting the parameters can help tune the server for particular workloads. Cloud currently exposes just those parameters which do not require a server restart to take effect.
+
+To access these settings, go to the Configuration tab in your Cloud formation and click "Change a Parameter." It will present a dropdown list of config parameters:
+
+.. image:: ../images/cloud-pg-conf.png
+
+The options are grouped by which server and system they control:
+
+* Coordinator PostgreSQL
+* Coordinator Inbound PgBouncer
```

Comment:
We will also allow Coordinator Outbound PgBouncer soon but we can add it once we enable it.